index.js

```diff
@@ -517,9 +517,16 @@ class Stream extends EventEmitter {
       if (opts.open) this._open = opts.open
       if (opts.destroy) this._destroy = opts.destroy
       if (opts.predestroy) this._predestroy = opts.predestroy
+      if (opts.signal) {
+        opts.signal.addEventListener('abort', this.abort.bind(this))
+      }
     }
   }
 
+  abort () {
```

Comment:
This should be an internal function so people don’t overwrite it. Ie function abort () somewhere in the bottom of the file

Reply:
Ah!

Reply:
pushed